sparse_ho/algo/implicit_forward.py

```diff
@@ -148,6 +148,7 @@ def get_only_jac(
             model.get_jac_obj(Xs, y, n_samples, sign_beta, dbeta, dual_var,
                               ddual_var, alpha))
         if use_stop_crit and i > 1:
-            if np.abs(objs[-2] - objs[-1]) < np.abs(objs[-1]) * tol_jac:
+            if np.abs(objs[-2] - objs[-1]) < np.abs(objs[-1]) * tol_jac \
```

Comment:
I think we try to avoid `\`,  you can use parenthesis isntead to break the line: 
```if ( .....   
        or ....):  
```

Reply:
yes that's right